### PR TITLE
micro: add --use-host-config to Fedora image

### PR DESCRIPTION
**Pull request by pkubatrh**
also move Fedora image to use f39 repos

### CODE REVIEW

**Comment by zmiklank** [comment]
[test]

File: 2.4-micro/Dockerfile.fedora [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM quay.io/fedora/fedora:rawhide AS build
 RUN mkdir -p /mnt/rootfs
 RUN MICRO_PKGS="coreutils-single glibc-minimal-langpack" && \
     INSTALL_PKGS="$MICRO_PKGS httpd-core mod_ssl findutils hostname nss_wrapper-libs fedora-logos-httpd" && \
-    dnf install --installroot /mnt/rootfs $INSTALL_PKGS --releasever 36 --setopt install_weak_deps=false --nodocs -y && \
+    dnf install --installroot /mnt/rootfs --use-host-config $INSTALL_PKGS --releasever 39 --setopt install_weak_deps=false --nodocs -y && \
```
Comment (phracek) [inline 2.4-micro/Dockerfile.fedora:6]: Only one question, why `releasever 39`?

**Review by phracek — COMMENTED** [review]
Only one nitpick

**Comment by phracek** [comment]
[test]

File: 2.4-micro/Dockerfile.fedora (hunk 1 above)
Comment (pkubatrh) [reply 2.4-micro/Dockerfile.fedora:6]: Because we are currently tracking rawhide

Comment (zmiklank) [reply 2.4-micro/Dockerfile.fedora:6]: Is it a goal for all of Fedora micro images to track rawhide?

**Review by zmiklank — COMMENTED** [review]
@pkubatrh thank you for the PR.

Could you please explain more in the commit message why  the --use-host-config option had to be added?

**Comment by pkubatrh** [comment]
Added some more details along with a link to dnf PR.

File: 2.4-micro/Dockerfile.fedora (hunk 1 above)
Comment (pkubatrh) [reply 2.4-micro/Dockerfile.fedora:6]: Not a goal but that is what we started with as the packages were not available anywhere else.
We can decide on pinning a specific Fedora version in a separate PR if that is something we want to do.

**Review by zmiklank — APPROVED** [review]

**Pull request merged**